src/evict/evict_page.c

```diff
@@ -656,7 +656,13 @@ __evict_review(
 		else if (!WT_IS_METADATA(session->dhandle)) {
 			LF_SET(WT_REC_UPDATE_RESTORE);
 
-			if (F_ISSET(cache, WT_CACHE_EVICT_SCRUB))
+			/*
+			 * Scrub if we're supposed to or toss it in sometimes
+			 * if we are in debugging mode.
+			 */
+			if (F_ISSET(cache, WT_CACHE_EVICT_SCRUB) ||
+			    (F_ISSET(cache, WT_CACHE_EVICT_DEBUG_MODE &&
```

Comment:
Need a closing ')' after the `F_ISSET`, not after the `__wt_random`.

Reply:
Yikes! The compiler failed me on that one. Fixed.